packages/components/package.json

```diff
@@ -46,6 +45,8 @@
     "postcss-cli": "^9.1.0",
     "postcss-import": "^14.1.0",
     "postcss-loader": "^7.0.0",
+    "react": "^18.1.0",
```

Comment:
I think I don't understand imports too well. Why do we need to have react 18 as a dependency, if it's also a peer dependency?

Reply:
We need it as a _dev_ dependency, since it should be installed for storybook etc. Here: https://stackoverflow.com/a/30454133